docs/CONTRIBUTING.md

```diff
@@ -4,6 +4,144 @@ Feel free to contribute through issues and/or pull requests. I will try my best
 
 The instructions below are taken from [Basssiiie's Typescript Template](https://github.com/Basssiiie/OpenRCT2-Simple-Typescript-Template), though the template is slightly modified from the original.
 
+## Adding more statistics
+
+This plugin offers easy extensibility for tracking new statistics. Look through `src/statistics/timeSpent.ts` to see an existing implementation.
+
+### Steps for designing your own statistic widget:
+
+1. Identify the stat you want to track & make sure there's a way to actually do that in the game.
+
+   - For instance, look through the `HookType` and `ActionType` types for ideas of what can be listened for/hooked into. For example, you can listen to just ride creation events by subscribing to `action.execute` and filter the events for those which are `ridecreate`.
+
+2. (optional) Create a type to describe the shape of your statistic you're going to track. This should be a single instance of the stat, not a group or array of stats. If you're tracking the amount of time played, this could be `type TimeSpendStat = number`. If you're doing something more complicated, an object could be useful, e.g.
```

Comment:
An instance of `TimeSpendStat` that wasn't caught. Which leads into one point I think would help maintaining this `CONTRIBUTING.md` file: There should probably be less code blocks.

Any time code is copy-pasted here, there's the potential this document will become outdated as changes are made. Perhaps having more instances of something along the lines of "See `subscribeToTimePassing` in [`timeSpent.ts`](../src/statistics/timeSpent.ts)" would help to reduce the amount of potential changes.

If simple enough, examples can perhaps be here (honestly, this example of `TimeSpendStat` was less a critisim of using direct copy-paste here and more an example of why this could cause problems in the future; it illistrates this point well), but most examples, especially longer function implementations, might be best served in their original files.